src/Data/Models/User.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace tcms.Data.Models
+{
+	public class User

```

Comment:
Please see how to define our own User entity [here](https://entityframework.net/knowledge-base/20104289/foreign-key-to-microsoft-aspnet-identity-entityframework-identityuser-).
That way you don't need regular "user" fields.